lib/commands/check-versions.js

```diff
@@ -0,0 +1,61 @@
+const Input = require('../utility/input');
+const Output = require('../utility/output');
+const Repository = require('../utility/repository');
+const AdmZip = require('adm-zip');
```

Comment:
@icc As far as I can see `archiver` only allows you to generate archives, not unzip/open them.